### PR TITLE
UIREQ-612: Create new request filter

### PR DESCRIPTION
**Pull request by pavelspichonak**
## Purpose
Create request level filter

## Screenshots
image_2021_12_24T11_30_33_859Z

## Refs
https://issues.folio.org/browse/UIREQ-612

### CODE REVIEW

File: src/components/RequestsFilters/RequestLevelFilter/RequestLevelFilter.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
+import React, {
+  useCallback,
+  useMemo,
+} from 'react';
+import { injectIntl } from 'react-intl';
```
Comment (Stanislau-Snapok) [inline src/components/RequestsFilters/RequestLevelFilter/RequestLevelFilter.js:5]: Better use `useIntl` hook i think

File: src/routes/RequestsRoute.js [hunk 2]
```diff
@@ -800,12 +800,16 @@ class RequestsRoute extends React.Component {
   }
 
   renderFilters = (onChange) => {
+    const { resources } = this.props;
+    const { titleLevelRequestsFeatureEnabled = false } = getTlrSettings(resources.configs.records[0]?.value);
+
     return (
       <RequestsFilters
         activeFilters={this.getActiveFilters()}
         resources={this.props.resources}
```
Comment (Stanislau-Snapok) [inline src/routes/RequestsRoute.js:809]: Can we here pass `resources` instead of `this.props.resources` since we are destructuring above?

**Review by Stanislau-Snapok — APPROVED** [review]

File: src/routes/RequestsRoute.js (hunk 2 above)
Comment (pavelspichonak) [reply src/routes/RequestsRoute.js]: Done

File: src/components/RequestsFilters/RequestLevelFilter/RequestLevelFilter.js (hunk 1 above)
Comment (pavelspichonak) [reply src/components/RequestsFilters/RequestLevelFilter/RequestLevelFilter.js]: My bad. Thanks.

**Comment by pavelspichonak** [comment]
@mkuklis @zburke could you please review?

**Review by mkuklis — APPROVED** [review]

**Pull request merged**